Xamarin.Essentials/Types/Contact.shared.cs

```diff
@@ -1,48 +1,59 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 
 namespace Xamarin.Essentials
 {
     public class Contact
     {
-        internal Contact(string name, IEnumerable<ContactPhone> phones, IEnumerable<ContactEmail> email)
+        public Contact()
+        {
+        }
+
+        public Contact(string name, IEnumerable<ContactPhone> phones, IEnumerable<ContactEmail> email)
         {
             Name = name;
             Emails = email?.ToList();
             Phones = phones?.ToList();
         }
 
-        public string Name { get; }
+        public string Name { get; set; }
 
-        public IReadOnlyList<ContactPhone> Phones { get; }
+        public List<ContactPhone> Phones { get; set; } = new List<ContactPhone>();
```

Comment:
I made this `List<T>` and make the type writeable to get ready for the later version that will support saving contacts. I went with list to be consistent with things like email attachments and sms recipients.